examples/sideinput-udf/src/main.rs

```diff
@@ -0,0 +1,46 @@
+use notify::{Watcher, RecursiveMode, Result};
+use numaflow::map::{Mapper,Message,start_uds_server,Datum};
+use std::path::Path;
+use tonic::{async_trait};
+use tokio::spawn;
+
+const DIR_PATH: &str = "/var/run/numaflow/sideinput.sock";
```

Comment:
why are we watching on `.sock` file? that file is for the UDS socket.

Reply:
I forgot to push the changes ,updated the path now